frontend/src/components/QCFlag.js

```diff
@@ -6,8 +6,8 @@ import {CheckOutlined, CloseOutlined} from "@ant-design/icons";
 const getFlagContent = (flags) => {
   return  (
     <div>
-      {Object.entries(flags).map(([name, value]) => {
-        return <p> {name} : {value ? "passed" : "failed"} </p>
+      {Object.entries(flags).map(([name, value], i) => {
+        return <p key={i.toString()}> {name} : {value ? "passed" : "failed"} </p>
```

Comment:
Does <p> tag have a key attribute ?

Reply:
Every React elements appears to have key.

Reply:
is this a react component ? As far as i know this is basic html dating back forever. I have not seen any reference where <p> has a "key" attribute. The closest i have found is "id" which may serve a similar role. most likely it does not exist and js just ignores it, but you should confirm.

Reply:
Oh right. In react, even HTML components have key. At least, according to the completion in VSCode.

Reply:
eheh it's possible there is some wrapper. it just feels overkill to add keys to component that do not really use them. It's no big deal either way.